Sources/ManageBans.php

```diff
@@ -268,6 +268,7 @@ function BanList()
 		'additional_rows' => array(
 			array(
 				'position' => 'top_of_list',
+				'class' => 'floatright',
```

Comment:
I would suggest removing the added `floatright` classes and editing the CSS code instead by adding after `.additional_rows`

```css
.additional_row::after {
    content: "";
    display: block;
    clear: both;
}
```

that way all elements of that type will be fixed and not only the ones where you had added the `floatright` class

Reply:
Done.